Documentation/gettingstarted/nodeport.rst

```diff
@@ -132,3 +125,16 @@ deployed to see the routing being performed correctly:
     $ ip r a 192.0.2.233 via <node-ip>
     $ curl 192.0.2.233:82
     <html><body><h1>It works!</h1></body></html>
+
+
+Limitations
+###########
+
+Similar to kube-proxy, if a pod on a node attempts to initiate a connection to
+an ``externalIP`` and the external IP is assigned to a node in the cluster, the
+connection to the service will be translated and forwarded to a backend.
+However, if a pod attempts to initiate a connection to an ``externalIP`` which
+does not belong to a node in the cluster, the service will *NOT* be translated.
```

Comment:
s/`does not belong to a node in the cluster`/`does not belong to the node where the connection is originated`

Reply:
It uses the IPcache identity == `HOST_ID` to determine this, isn't that every node in the cluster?

(Inspection of testing cluster seems to indicate this is the case)

Reply:
Oh I had the impression it was the local host. Ok, let's leave it as is and we need to change it once #9165 is merged